Library/ViewModels/ShareViewModel.swift

```diff
@@ -246,14 +241,14 @@ private func twitterInitialText(forShareContext shareContext: ShareContext) -> S
 }
 
 private func shareComposeController(forShareContext shareContext: ShareContext, serviceType: String)
-  -> SLComposeViewController {
+  -> SLComposeViewController? {
 
     let controller = SLComposeViewController(forServiceType: serviceType)
 
-    controller.addURL(shareUrl(forShareContext: shareContext))
+    shareUrl(forShareContext: shareContext).doIfSome { controller?.add($0) }
 
     if serviceType == SLServiceTypeTwitter {
-      controller.setInitialText(twitterInitialText(forShareContext: shareContext))
+      controller?.setInitialText(twitterInitialText(forShareContext: shareContext))
```

Comment:
`SLComposeViewController.init` is now failable so had to do some dancing around that